app/controllers/v0/ppiu_controller.rb

```diff
@@ -32,5 +31,33 @@ def ppiu_params
         :financial_institution_routing_number
       )
     end
+
+    def current_user_email
+      current_user.vet360_contact_info&.email&.email_address || current_user.email
+    end
+
+    def pay_info
+      @pay_info ||= EVSS::PPIU::PaymentAccount.new(ppiu_params)
+    end
+
+    def validate_pay_info
+      unless pay_info.valid?
+        Raven.tags_context(validation: 'direct_deposit')
+        raise Common::Exceptions::ValidationErrors, pay_info
+      end
+    end
+
+    def send_confirmation_email
+      if current_user_email
+        DirectDepositMailer.build(current_user_email, params[:ga_client_id]).deliver_now
+      else
+        log_message_to_sentry(
```

Comment:
Checked with stakeholder.  If no email is available, we will just log a message in Sentry for now.